packages/node_modules/samples/browser-plugin-meetings/app.js

```diff
@@ -526,12 +526,21 @@ document.getElementById('updateAudio').addEventListener('click', () => {
 
   // TODO: change this to check previous media values
   if (media.sendAudio) {
-    meeting.getMediaStreams(media, {audio: audioInputSelect.value ? {deviceId: {exact: audioInputSelect.value}} : media.sendAudio})
-      .then(([localStream]) => meeting.updateAudio({
-        stream: localStream,
-        sendAudio: media.sendAudio,
-        receiveAudio: media.receiveAudio
-      }));
+    let audio = true;
```

Comment:
So the variable `audio` can be either an object or a boolean. That seems confusing.

Reply:
I had to dive deep to find the answer to this. This property eventually gets sent to [getUserMedia](https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia) which you can specify `true` to allow the browser to pick an audio device, or you can give it a specific device id in the form of the object shape defined in line 532. 

So, to answer your question, "yes" 🤣

Reply:
I added some comments to the sample app code as well about this. Might be helpful for others using our sample.